news/2020-07-23-summer-theme-park-fanart-contest-results.md

```diff
@@ -0,0 +1,105 @@
+
+Either way, you're coming along, and it isn't surprising in the slightest that this picked up third place overall. Simply superb.
+
+[![](https://assets.ppy.sh/contests/107/banners/xxxxxx2800.jpg)](https://assets.ppy.sh/contests/107/winners/xxxxxx2800.png)
+
+Scoring **4877** votes, [**xxxxxx2800**](https://osu.ppy.sh/users/4084853)'s take on Maria, pippi and Chirou enjoying some candy at an all-gamemodes theme park impressed voters for sure.
+
+[![](https://assets.ppy.sh/contests/107/banners/KrisyYy.jpg)](https://assets.ppy.sh/contests/107/winners/KrisyYy.png)
+
+Edging in as the last of the badge-winning finalists with **4791** votes, [**KrisyYy**](https://osu.ppy.sh/users/7266506)'s take on a more maturely-fashioned Mocha and inexplicable catgirl pippi enjoying the seaside view from atop a ferris wheel was well ahead of the rest of the pack, and for good reason.
+
+It may be hard to take your eyes off the mascots, but a little attention diverted to the scenic background reveals a stunning amount of attention to detail, including things like the little rocks on the beach, the wash of the coastline and even the specular refraction off the ocean and ride compartment. Awesome stuff.
+
+[![](https://assets.ppy.sh/contests/107/banners/McFriedFries.jpg)](https://assets.ppy.sh/contests/107/winners/McFriedFries.png)
+
+Coming in sixth place with **4646** votes, [**McFriedFries**](https://osu.ppy.sh/users/12694139) wowed with their summertime botanical pippi enjoying a bit of rest with a plunging rollercoaster doing its thing in the background.
+
+[![](https://assets.ppy.sh/contests/107/banners/Ariisha.jpg)](https://assets.ppy.sh/contests/107/winners/Ariisha.png)
+
+[**Ariisha**](https://osu.ppy.sh/users/6246943) nabbed **4543** votes for a superbly stylized take on a whole bunch of our mascots enjoying time at a splash park, including Yuzu being harassed by the lifeguard for going into the pool without a shirt on.
+
+The connotations of that need not be said, but I doubt they surprise anyone at this point.
+
+[![](https://assets.ppy.sh/contests/107/banners/XShadowGirlX.jpg)](https://assets.ppy.sh/contests/107/winners/XShadowGirlX.png)
+
+[**XShadowGirlX**](https://osu.ppy.sh/users/11596939) scored **3963** votes for a gentle nighttime portrayal of pippi enjoying some cotton candy before a skill-testing machine filled with all sorts of community icons, including (but not limited to) the Taikonater, Sotarks, WhiteCat, Airman, Mocha and a strip of bacon. Crispy.
+
+[![](https://assets.ppy.sh/contests/107/banners/TofuMang.jpg)](https://assets.ppy.sh/contests/107/winners/TofuMang.png)
+
+Long-time crowd favourite [**TofuMang**](https://osu.ppy.sh/users/4817223) returns to scoop up **3922** votes with his delightfully fantastical take on pippi enjoying a most marvellous carousel ride, complete with a sky whale, and two hot Airman balloons.
```

Comment:
There's only one Airman hot air balloon, the other being Taikonator.

Reply:
this is meant as a semi-literal play on the term 'hot air balloon' to describe both instead of the one that's just airman